source/locales.tex

```diff
@@ -4689,9 +4689,9 @@
     class InputIterator = istreambuf_iterator<charT> >
   class money_get : public locale::facet {
   public:
-    typedef charT               char_type;
-    typedef InputIterator       iter_type;
-    typedef basic_string<charT> string_type;
+    using   char_type = charT;
```

Comment:
Left-align is good.
-------- Original message --------From: Alisdair Meredith <notifications@github.com> Date: 4/16/2016  9:34 AM  (GMT-08:00) To: cplusplus/draft <draft@noreply.github.com> Subject: Re: [cplusplus/draft] [18-30] Replace typedefs with alias (#704) 
In source/locales.tex:
> @@ -4689,9 +4689,9 @@
>      class InputIterator = istreambuf_iterator<charT> >
>    class money_get : public locale::facet {
>    public:
> -    typedef charT               char_type;
> -    typedef InputIterator       iter_type;
> -    typedef basic_string<charT> string_type;
> +    using   char_type = charT;

It is a bit cute. I largely threw it in to see the reaction, as we work out how to align the new syntax.  I like it, but suspected it might not be the preferred choice.  Unless I hear support, I will left-align any remaining right-aligned identifiers, but retain the alignment on the '='.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub